client/src/views/OrgsList.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import makeStyles from '@mui/styles/makeStyles';
+import Card from '@mui/material/Card';
+import TodayOutlined from '@mui/icons-material/TodayOutlined';
+import RoomOutlined from '@mui/icons-material/RoomOutlined';
+import Grid from '@mui/material/Grid';
+
+import type { Theme } from '@mui/material/styles';
+
+import type { Asset } from '../types';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  needsAndOffersSub: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    flexWrap: 'wrap',
+    alignItems: 'stretch',
+  },
+  card: {
+    marginBottom: '30px',
+    '&:not(:last-child)': {
+      marginRight: '20px',
+    },
+    flex: '0 0 30%',
+  },
+  cardImg: {
+    borderRadius: '5px',
+    margin: '10%',
+    maxWidth: '80%',
+  },
+  needsAndOffersHeader: {
+    textAlign: 'left',
+    paddingBottom: '20px',
+  },
+  needsAndOffers: {
+    padding: '10%',
+  },
+  cardText1: {
+    padding: '0 10%',
+  },
+  cardText2: {
+    padding: '0 10% 10%',
+  },
+}));
+
+type Props = {
+  assets: Asset[];
+  headerContentRight?: JSX.Element;
+  headerText: string;
+};
+
+function AssetsList(props: Props): JSX.Element {
+  const classes = useStyles();
+  const { assets, headerContentRight, headerText } = props;
+  if (!assets) return <> </>;
+
+  return (
+    <>
+      <Grid container item justifyContent="space-between">
+        <Typography variant="h4" component="h4" className={classes.needsAndOffersHeader}>
+          {headerText}
+        </Typography>
+        {headerContentRight ?? null}
+      </Grid>
+      <div className={classes.needsAndOffersSub}>
+        {assets.map((asset) => (
+          <NavLink to={`/asset/${asset.id}`} key={asset.id} className={classes.card}>
+            <Card variant="outlined">
+              <img
+                src={
+                  asset?.imgUrls?.[0] ??
+                  'https://optinmonster.com/wp-content/uploads/2019/09/nonprofit-newsletter.png'
+                }
+                className={classes.cardImg}
+                alt={asset.title}
+              />
+              <Typography variant="h6" component="h4" className={classes.cardText1}>
+                {asset.title} {asset.categories ? asset.categories[0] : null}
+              </Typography>
+              <div className={classes.cardText2}>
+                <RoomOutlined />
+                {asset.location}
+                <TodayOutlined />
+                {new Date(asset.datePosted).toLocaleDateString()}
+              </div>
+            </Card>
```

Comment:
if you `cd server` and then `npm run seed:drop` you'll get some orgs in your DB, then your fetch orgs will work, and you should see actual data in this component. lmk if that's not the case

once that happens if you can make this card look nice for orgs that'd be great. but don't spend too much time on it because we're still waiting to see what design wants it to look like. you can also just show the org json